crates/valence/src/instance.rs

```diff
@@ -363,6 +364,32 @@ impl Instance {
         );
     }
 
+    /// Plays a sound effect at the given position in the world. The sound
+    /// effect is audible to all players in the instance with the
+    /// appropriate chunk in view.
+    pub fn play_sound(
+        &mut self,
+        sound: Sound,
+        category: SoundCategory,
+        position: impl Into<DVec3>,
+        volume: f32,
+        pitch: f32,
+    ) {
+        let position = position.into();
+
+        self.write_packet_at(
+            &SoundEffect {
+                id: sound.to_id(),
+                category,
+                position: (position * 8.0).as_ivec3().into(),
+                volume,
+                pitch,
+                seed: rand::random(),
```

Comment:
We might want to add `play_sound_seeded` variants of this method that take the seed as a parameter in the future.